velox/exec/tests/AggregationFuzzer.cpp

```diff
@@ -152,6 +153,14 @@ class AggregationFuzzer {
       std::vector<std::string> names,
       std::vector<TypePtr> types);
 
+  // Generate a RowVector of the given types of children with an additional
+  // child named "id" of BIGINT identifiers that differentiates every row.
+  // Identifier starts from 0. This additional input vector is needed for
+  // result verification of window aggregations.
+  std::vector<RowVectorPtr> generateInputDataWithId(
```

Comment:
nit: here ID is really a row number, hence, consider renaming to generateInputDataWithRowNumber and use "row_number" as column name